sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/serializer/ObjectSerializer.java

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.experimental.serializer;
+
+/**
+ * Generic interface for object serialization.
+ */
+public interface ObjectSerializer {
+}
```

Comment:
I guess we can start with the serialize and deserialize methods that is used in the latest avro PR. Might help unblock @arerlend  too.

Reply:
That is one of my goals with this change, to unblock testing of the serialization interfaces. This will require package import changes once these changes make their way into Azure Core proper.